pkgs/applications/editors/vscode/generic.nix

```diff
@@ -134,7 +134,7 @@ in
       startupNotify = true;
       startupWMClass = shortName;
       categories = [ "Utility" "TextEditor" "Development" "IDE" ];
-      mimeTypes = [ "text/plain" "inode/directory" ];
+      mimeTypes = [ "text/plain" ];
```

Comment:
Why keep the `text/plain` association?

Reply:
vscode is a text editor?

Reply:
oh, I guess they removed that upstream too.